docs/pages/loaders.vue

```diff
@@ -11,6 +11,99 @@
       </p>
     </DocsPageSection>
 
+    <DocsPageSection title="Linear loaders" anchor="#linear loaders">
```

Comment:
We have pages specifically for the `KLInearLoader` and `KCircularLoader`, so I would expect most of this content to be in those. I think the `Loaders` page should focus on general patterns and guidance about when to choose one over the other